src/main/java/core/basesyntax/service/ReportCreatorImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class ReportCreatorImpl implements ReportCreator {
+    @Override
+    public String createReport() {
+        StringBuilder stringBuilder = new StringBuilder("fruit,quantity");
```

Comment:
Create constant for header "fruit,quantity" and for delimiter ","

Reply:
not fixed